### PR TITLE
Improve the English

### PR DESCRIPTION
**Pull request by brianteeman**
Simple PR to improve the English and readability of one string.

### Before
MOD_SAMPLEDATA_CONFIRM_START="Proceeding will install a sample data set into your Joomla. This process can't be reverted once done."

### After
MOD_SAMPLEDATA_CONFIRM_START="Proceeding will install a sample data set into your Joomla web site. This process can't be reverted once done."

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 2b281b7bd461283b71e1527d03534b442fa2b9dd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18495">issues.joomla.org/tracker/joomla-cms/18495</a>.</sub>

File: administrator/language/en-GB/en-GB.mod_sampledata.ini [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ; Note : All ini files need to be saved as UTF-8
 
 MOD_SAMPLEDATA="Sample Data"
-MOD_SAMPLEDATA_CONFIRM_START="Proceeding will install a sample data set into your Joomla. This process can't be reverted once done."
+MOD_SAMPLEDATA_CONFIRM_START="Proceeding will install a sample data set into your Joomla web site. This process can't be reverted once done."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.mod_sampledata.ini]: one word `website`.

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.mod_sampledata.ini]: oops broke the rules that I wrote

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 26ed5a91c4177bdac337a02384fc0620461957d7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18495">issues.joomla.org/tracker/joomla-cms/18495</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 26ed5a91c4177bdac337a02384fc0620461957d7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18495">issues.joomla.org/tracker/joomla-cms/18495</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**